shell/platform/darwin/ios/framework/Source/accessibility_bridge.mm

```diff
@@ -13,17 +14,54 @@
 FLUTTER_ASSERT_NOT_ARC
 
 namespace flutter {
+namespace {
+
+FlutterViewController* _Nullable GetFlutterViewControllerForView(UIView* view) {
+  // There is no way to get a view's view controller in UIKit directly, this is
```

Comment:
Looks like platformviewios right above it has a FlutterViewController anyway. We can just refactor this private constructor to take a FVC if needed.

Reply:
Yea, i can look into it.  i'll split it into a different PR to keep the bug fix and the refactor separate.

Reply:
LG

Reply:
https://github.com/flutter/engine/pull/18800